Sources/ClientRuntime/Retries/SDKRetryer.swift

```diff
@@ -95,3 +93,16 @@ public class SDKRetryer: Retryer {
         }
     }
 }
+
+extension AwsCommonRuntimeKit.RetryStrategy {
+    convenience init(options: RetryOptions, eventLoopGroup: EventLoopGroup) throws {
+       try self.init(
+            eventLoopGroup: eventLoopGroup,
+            initialBucketCapacity: options.initialBucketCapacity,
+            maxRetries: options.maxRetries,
+            backOffScaleFactor: options.backOffScaleFactor,
+            jitterMode: options.jitterMode.toCRTType(),
+            generateRandom: nil // we should pass in the options.generateRandom but currently it fails since the underlying closure is a c closure
```

Comment:
This maintains the existing behavior where we just pass in `nil` to generateRandom since it is a c closure. An issue has been filed with CRT to change the type exposed so that we can pass in a swift closure. Otherwise, we would need to expose the c closure type to consumers which doesn't make sense as it feels like an implementation detail leaking out.